tools/release.sh

```diff
@@ -58,7 +58,7 @@ if [ -z "$(git config --get user.name)" ]; then
 fi
 
 if [ -z "$(git config --get user.email)" ]; then
-    git config user.email "<dracutng@gombos.dev>"
+    git config user.email "<dracut>"
```

Comment:
```suggestion
    git config user.email "<dracutng@localhost.localdomain>"
```

Reply:
Don't we have any contact address that we could use here?